### DISCUSSION TITLE
Avvio domotica casa nuova, qualche consiglio.

### DISCUSSION THREAD

**Discussion by sandrollo** [indomus/forum#3281] [GENERALE - Discussioni generiche]
Ciao a tutti, 
sono nuovo di questo forum e anche per quello che riguarda la domotica in generale, sono però un sistemista e so programmare, non avrei quindi problemi a creare una soluzione personalizzata.

Mi sto addentrando nel mondo dei dispositivi di domotica per la mia nuova abitazione, avviando il progetto da zero vorrei scegliere una soluzione che mi permetta, inizialmente di avere un sistema funzionante e con il tempo di farlo crescere con nuove funzioni e personalizzazioni.

Al momento mi interessa poter mettere le valvole termostatiche, eventualmente dei sensori di temperatura nelle stanze con più di un termosifone, qualche sensore di apertura finestre e di controllare la caldaia a condensazione di conseguenza.
Attualmente c'è un solo termostato, connesso direttamente alla caldaia in una casa di due piani con una sola zona (la casa è anni 60 e la suddivisione in zone non era ben contemplata all'epoca) 

Ho scartato la soluzione TadoX per il costo più elevato e perché mi sembra un sistema abbastanza chiuso e mi sto orientando su sonoff con ihost.

Ho solo un dubbio sull'interfacciamento verso la caldaia BAXI, sembra che non esista un componente con interfaccia opentherm quindi mi devo orientare su uno switch a contatto pulito on/off, possibilmente zigbee, ma non saprei cosa scegliere ... in teoria aqara T1 e T2 dovrebbero andare bene ma non ne sono sicuro, e non sono convinto di poterli gestire da dentro ihost

Avete qualche suggerimento o consiglio?
Grazie a tutti

Sandro

**Comment by indomus** [comment]
> Avete qualche suggerimento o consiglio? Grazie a tutti

Il migliore che possiamo darti è quello di avere ben CHIARO lo scenario smart home che, nel 99% dei casi, sfugge. Prenditi 20 minuti per leggere [questo breve percorso](https://indomus.it/nozioni-di-base/). Al termine dovresti avere ben chiari certi concetti, e dovresti aver anche optato per Home Assistant, il quale dovrebbe essere il primo mattone da mettere in opera, prima di qualsiasi altra cosa.
Per interfacciarti OpenTherm puoi usare la linea [tado](https://amzn.to/3Oc08Le) o altre, che cmq gestiscono anche via contatto pulito. Volendo sì, puoi anche semplicemente usare un attuatore a contatto pulito da integrare a Home Assistant, tramite il quale creare un termostato virtuale in associazione a un qualunque termometro wireless anch'esso integrato.. però sui riscaldamenti, dai retta, usa una linea di prodotti. Avventurarsi a pianificare l'apertura delle singole valvole in modo programmatico diventa un discreto casino, fidati sulla parola.

**Comment by sandrollo** [comment]
Grazie della risposta, la guida l'ho letta e riletta come anche le vostre recensioni, complimenti per il lavoro.

Mi ero infatti orientato su ihost proprio per poter utilizzare docker e avviare homeassistant. Avrei anche possibilità di lanciare un container homeassistant direttamente sul router e in un secondo momento migrare su altro hw più potente

però pensavo però ad ihost, abbinato a un paio di NSPanel, per la tranquillità di avere un sistema completo tuttofare, e anche abbastanza carino, e di implementare homeassitant in docker, collegando i dispositivi sonoff (e da quello che ho capito anche altri con Zigbee2CUBE) via eWeLink Smart Home. ma magari mi sbaglio e tutto ciò non è possibile.

Si, tado X sarebbe pienamente compatibile, era la prima soluzione individuata, ma presenta dei costi più alti per l'intero sistema di solo riscaldamento.

mi sto anche leggendo l'openboard e la news di oggi, così approfondisco ancora

Grazie
Sandro

**Reply by KC-inDomus** [reply]
sei sicuro di voler usare ihost? io ci penserei 3 volte, e poi prenderei un mini pc.
